README.rst

```diff
@@ -389,6 +389,17 @@ configuration must list one or more nodes (under the configuration key
 object of libpq key: value connection info pairs or libpq connection string
 or a ``postgres://`` connection uri.
 
+``basebackup_chunk_count`` (default ``5``)
```

Comment:
I'd call this something else -- this name sounds like it would define the total number of chunks a basebackup is split to.  Use `basebackup_chunks_in_progress` or something like that?